docs/language/operators/from.md

```diff
@@ -159,13 +164,32 @@ zed -lake example query -f text 'from * | count()'
 _Join the data from multiple pools_
 ```mdtest-command
 zed -lake example query -z '
-  from (
-    pool coinflips => sort flip
-    pool numbers => sort number
-  ) | join on flip=number word'
+  from coinflips | sort flip
+  | join (
+    from numbers | sort number
+  ) on flip=number word'
+```
+=>
+```mdtest-output
+{flip:1,result:"heads",word:"one"}
+{flip:2,result:"tails",word:"two"}
+```
+
+_Use `pass` to combine our join output with data from yet another source_
+```mdtest-command
+zed -lake example query -z '
+  from coinflips | sort flip
+  | join (
+    from numbers | sort number
+  ) on flip=number word
+  | from (
+      pass
+      pool coinflips@trial => c:=count() | yield "There were ${int64(c)} flips"
```

Comment:
I think these should be indented two spaces from `| from (` (just as `from numbers | sort number` is indented two spaces from `| join (`).
```suggestion
  | from (
    pass
    pool coinflips@trial => c:=count() | yield "There were ${int64(c)} flips"
```

Reply:
@nwt: Yeah, you're right. I don't know what I was thinking. I reversed it again.